From: Bryan Gardiner <bog@khumba.net>
Date: 2023-06-30

NetworkManager should not allow everyone to modify system connections without
authentication.  Fix Gentoo bug #905668 by neutering the modify_system build
option.

diff -ru NetworkManager-1.42.6/meson.build NetworkManager-1.42.6-b/meson.build
--- NetworkManager-1.42.6/meson.build	2023-04-19 23:44:36.000000000 -0700
+++ NetworkManager-1.42.6-b/meson.build	2023-06-30 08:40:51.296672198 -0700
@@ -508,7 +508,7 @@
 endif
 config_h.set_quoted('NM_CONFIG_DEFAULT_MAIN_AUTH_POLKIT', config_auth_polkit_default)
 
-enable_modify_system = get_option('modify_system')
+enable_modify_system = false  #get_option('modify_system')
 
 polkit_agent_helper_1_path = get_option('polkit_agent_helper_1')
 foreach p : [ '/usr/libexec/polkit-agent-helper-1',
